app/models/account_user.rb

```diff
@@ -0,0 +1,3 @@
+class AccountUser < ActiveRecord::Base
+  enum role: [:user, :admin]
```

Comment:
sorry, I meant the fact that the account_users table has `user_id` and `account_id` but the model doesn't have those associations.